src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -137,7 +137,7 @@ public enum LossFunctionType
         /// <summary>
         /// Options for the <see cref="MatrixFactorizationTrainer"/> as used in [MatrixFactorization(Options)](xref:Microsoft.ML.RecommendationCatalog.RecommendationTrainers.MatrixFactorization(Microsoft.ML.Trainers.MatrixFactorizationTrainer.Options)).
         /// </summary>
-        public sealed class Options
+        public sealed class Options : TrainerInputBaseWithLabel
```

Comment:
This now adds a `FeatureColumnName` property to this class. I don't it gets used, correct? That may be confusing to users.

Reply:
`TrainerInputBaseWithLabel` adds `LabelColumnName` to class, which is already exists in `Options` class

Reply:
It _also_ adds a `FeatureColumnName` because it inherits from `TrainerInputBase`

https://github.com/dotnet/machinelearning/blob/edfd10f131abe97a7f7bb25c0070843a67b8b660/src/Microsoft.ML.Data/Training/TrainerInputBase.cs#L29-L33